### PR TITLE
[8.x] Adding tests to ValidatePostSize

### PR DESCRIPTION
**Pull request by komen205**
Adding more code coverage to ValidatePostSize.php

### CODE REVIEW

File: tests/Integration/Http/ResourceTest.php [hunk 1]
```diff
@@ -989,6 +993,15 @@ public function work()
         ], $results);
     }
 
+    public function testPostTooLargeException()
+    {
+        $this->expectException(PostTooLargeException::class);
+
+        $request = Mockery::mock(Request::class,['server'=>['CONTENT_LENGTH'=>'99999999999999999']]);
```
Comment (GrahamCampbell) [inline tests/Integration/Http/ResourceTest.php:1000]: This test won't work on 32 bit systems. Please choose a value smaller than 2147483647 to use for the content length, there.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: tests/Integration/Http/ResourceTest.php (hunk 1 above)
Comment (komen205) [reply tests/Integration/Http/ResourceTest.php]: I used the number 2147483640 which is < than 2147483647, should work.
Unfortunately, getPostMaxSize is protected and I'm not even sure it would work in a test. The number should work now. 
@GrahamCampbell

File: tests/Integration/Http/ResourceTest.php [hunk 2]
```diff
@@ -51,7 +50,8 @@ public function testResourcesMayBeConvertedToJson()
         });
 
         $response = $this->withoutExceptionHandling()->get(
-            '/', ['Accept' => 'application/json']
+            '/',
+            ['Accept' => 'application/json']
```
Comment (GrahamCampbell) [inline tests/Integration/Http/ResourceTest.php]: please revert all these unrelated changes

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: tests/Integration/Http/ResourceTest.php (hunk 2 above)
Comment (komen205) [reply tests/Integration/Http/ResourceTest.php]: My bad I used my VSCode formatted, should be good in the last commit #38332
